invenio_records_rest/links.py

```diff
@@ -29,7 +29,7 @@
 from .proxies import current_records_rest
 
 
-def default_links_factory(pid):
+def default_links_factory(pid, **kwargs):
```

Comment:
As you mentioned, this seems to be the only breaking change, considering that because of the previous "non-kwargs" signature, overlay link factories where defined having in mind the single argument definition...

Would it be possible to have this work for old functions (eg. [through inspection](https://stackoverflow.com/a/33029621)) but trigger an annoying warning?

*(PS: We will, of course, update our link factories on Zenodo, but I'm not sure how other people handle package upgrades)*